dz_1/dz_1_2.py

```diff
@@ -0,0 +1,19 @@
+"""
+Задача 4: Петя, Катя и Сережа делают из бумаги журавликов. Вместе они сделали S журавликов. 
+Сколько журавликов сделал каждый ребенок, если известно, что 
+Петя и Сережа сделали одинаковое количество журавликов, а 
+Катя сделала в два раза больше журавликов, чем Петя и Сережа вместе?
+*Пример:*
+6 -> 1  4  1
+24 -> 4  16  4
+60 -> 10  40  10
+"""
+
+total_quantity = int(input("Введите сколько всего сделали журавликов дети: "))
+one_quantity = int(total_quantity/6)
```

Comment:
Тут лучше применять целочисленное деление. Результат тот же, но запись более короткая
```suggestion
one_quantity = total_quantity // 6
```